libs/net/entities/TonePool.cs

```diff
@@ -27,7 +27,7 @@ public class TonePool : BaseType<int>
 
     public TonePool(string name, User owner) : base(name)
     {
-        this.OwnerId = owner?.Id ?? throw new ArgumentNullException(nameof(owner));
+        if (owner != null) this.OwnerId = owner.Id;
```

Comment:
You can't do this `OwnerId` is a required property.  It cannot default to `0`.

Reply:
The same comment as above, thanks